pkg/epp/util/podutils/podutils.go

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
nit: remove the util suffix from both the directory and file names and the pkg name, it is already under `util/` pkg

Reply:
I was looking on the following as an example for the dir+file names:
https://github.com/kubernetes/kubectl/blob/master/pkg/util/podutils/podutils.go
(not using it since I don't want to reference kubectl).

but np, changing as requested.